### PR TITLE
cloud: add release notes 20240604

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

cloud: add release notes 20240604

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## June 4, 2024
+
+**General changes**
+
+- Introduce the Replication Group feature (beta) for disaster recovery of [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters deployed on AWS.
+
+    You can now use the new Replication Group feature for enhanced disaster recovery on TiDB Dedicated clusters deployed on AWS. This feature allows databases to be replicated from one cluster to another, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new Replication Group and assigning databases to it. By replicating databases with Replication Groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md]:
```suggestion
    You can now use the new Replication Group feature for enhanced disaster recovery on TiDB Dedicated clusters deployed on AWS. This feature allows databases to be replicated from one cluster to another, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new replication group and assigning databases to the group. By replicating databases with replication groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## June 4, 2024
+
+**General changes**
+
+- Introduce the Recovery Group feature (beta) for disaster recovery of [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters deployed on AWS.
+
+    You can now use the new Recovery Group feature for enhanced disaster recovery on TiDB Dedicated clusters deployed on AWS. This feature allows databases to be replicated from one cluster to another, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new Recovery Group and assigning databases to it. By replicating databases with Recovery Groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    This feature allows you to replicate your databases between TiDB Cloud Dedicated clusters, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new replication group and assigning databases to the group. By replicating databases with replication groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-release-notes.md:17]: The deleted sentence overlaps slightly with what's already stated in line 15.

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,18 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## June 4, 2024
+
+**General changes**
+
+- Introduce the Recovery Group feature (beta) for disaster recovery of [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters deployed on AWS.
+
+    This feature enables you to replicate your databases between TiDB Cloud Dedicated clusters, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new recovery group and assigning databases to the group. By replicating databases with recovery groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    This feature enables you to replicate your databases between TiDB Dedicated clusters, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new recovery group and assigning databases to the group. By replicating databases with recovery groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
```

**Comment by yahonda** [comment]
https://github.com/pingcap/docs/blob/a05dc31ed34c8b26c41b049c9cb3718530267845/tidb-cloud/serverless-limitations.md?plain=1#L53 also needs updated.

**Comment by Oreoxmt** [comment]
> https://github.com/pingcap/docs/blob/a05dc31ed34c8b26c41b049c9cb3718530267845/tidb-cloud/serverless-limitations.md?plain=1#L53
> 
> also needs updated. I mean to say this line can be removed.

@yahonda This document will be updated in https://github.com/pingcap/docs/pull/17676

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,24 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**General changes**
+
+- Introduce the Recovery Group feature (beta) for disaster recovery of [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters deployed on AWS.
+
+    This feature enables you to replicate your databases between TiDB Dedicated clusters, ensuring rapid recovery in the event of a regional disaster. If you are in the `Project Owner` role, you can enable this feature by creating a new recovery group and assigning databases to the group. By replicating databases with recovery groups, you can improve disaster readiness, meet stricter availability SLAs, and achieve more aggressive Recovery Point Objectives (RPO) and Recovery Time Objectives (RTO).
+
+    For more information, see [Get started with recovery groups](/tidb-cloud/recovery-group-get-started.md).
+
+- [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) billing and metering (beta) for the columnar storage [TiFlash](/tiflash/tiflash-overview.md).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:21]: @qiancai PTAL

Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:21]:
```suggestion
- Introduce billing and metering (beta) for the [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) columnar storage [TiFlash](/tiflash/tiflash-overview.md).
```

**Pull request merged**